src/frontend/src/pages/CreateAccountPage.vue

```diff
@@ -39,6 +39,8 @@
       id="create-account-button"
       dest=""
       label="Create Account"
+      size="small"
+      type="default"
```

Comment:
can you add the password input type? should be a quick fix

Reply:
Forgot about that! Fixed